ingest/de_utils.py

```diff
@@ -0,0 +1,85 @@
+"""Utilities for differential expression
+"""
+
+import re
+
+
+# def get_size(metric):
```

Comment:
How come these functions are commented out rather than deleted? Or is the comment below "# TODO: Delete this function once UI can handle flexible metrics" related?

Reply:
Ah, I should've clarified.  The commented-out code here would mostly be uncommented when that TODO is done.